tests/TestCase/Controller/Component/ApiCacheComponentTest.php

```diff
@@ -0,0 +1,251 @@
+<?php
+declare(strict_types=1);
+
+namespace BEdita\WebTools\Test\TestCase\Controller\Component;
+
+use BEdita\SDK\BEditaClient;
+use BEdita\WebTools\ApiClientProvider;
+use BEdita\WebTools\Controller\Component\ApiCacheComponent;
+use Cake\Cache\Cache;
+use Cake\Controller\ComponentRegistry;
+use Cake\Core\Configure;
+use Cake\TestSuite\TestCase;
+
+/**
+ * {@see \Bedita\WebTools\Controller\Component\ApiCacheComponent} Test Case
+ *
+ * @coversDefaultClass \Bedita\WebTools\Controller\Component\ApiCacheComponent
+ */
+class ApiCacheComponentTest extends TestCase
+{
+    /**
+     * Test subject
+     *
+     * @var \App\Controller\Component\ApiCacheComponent
+     */
+    public $ApiCache;
+
+    /**
+     * setUp method
+     *
+     * @return void
+     */
+    public function setUp(): void
+    {
+        parent::setUp();
+        $registry = new ComponentRegistry();
+        $this->ApiCache = new ApiCacheComponent($registry);
+    }
+
```

Comment:
Add a `tearDown()` method to avoid using same mocked API client on other unit tests:

```php
    public function tearDown(): void
    {
        parent::tearDown();
        ApiClientProvider::setApiClient(null);
    }
```